include/clangmetatool/collectors/find_calls.h

```diff
@@ -0,0 +1,55 @@
+#ifndef INCLUDED_CLANGMETATOOL_COLLECTORS_FIND_CALLS_H
+#define INCLUDED_CLANGMETATOOL_COLLECTORS_FIND_CALLS_H
+
+#include <clang/Frontend/CompilerInstance.h>
+#include <clang/ASTMatchers/ASTMatchers.h>
+#include <clang/ASTMatchers/ASTMatchFinder.h>
+#include <clang/ASTMatchers/ASTMatchersInternal.h>
+#include <clang/AST/Expr.h>
```

Comment:
I think includes for those headers is not quite right, please consider running `include-what-you-use`